test/appium/tests/medium/test_chats_m.py

```diff
@@ -996,3 +996,102 @@ def test_block_user_from_1_1_chat_header_check_mentions_and_push_notification_se
+        self.public_key_1, self.default_username_1 = self.home_1.get_public_key_and_username(return_username=True)
+        self.public_key_2, self.default_username_2 = self.home_2.get_public_key_and_username(return_username=True)
+        [home.tap_mutual_cr_switcher() for home in (self.home_1, self.home_2)]
+        [home.home_button.click() for home in (self.home_1, self.home_2)]
+
+    @marks.testrail_id(702375)
+    def test_mutual_cr_unable_send_messages_if_users_not_contacts(self):
+        self.home_1.just_fyi("Creating 1-1 chats")
+        self.chat_1 = self.home_1.add_contact(self.public_key_2, add_in_contacts=False)
+        if self.chat_1.chat_message_input.is_element_displayed():
+            self.errors.append('Input field is displayed in chat with not a contact')
+        if not self.chat_1.contact_request_button.is_element_displayed():
+            self.errors.append('Send contact request button is not displayed in chat with not a contact')
+        self.errors.verify_no_errors()
+
+    @marks.testrail_id(702376)
+    def test_mutual_cr_send_and_accept_cr(self):
+        self.home_1.home_button.click()
+
+        self.home_1.just_fyi('Entering 1-1 chat and sending contact request')
+        self.chat_1 = self.home_1.add_contact(self.public_key_2, add_in_contacts=False)
+        self.chat_1.send_contact_request("Hi, please add me to contacts")
+
+        self.home_1.just_fyi('Check pending request is displayed, input field still disabled')
+        if not self.chat_1.element_by_text('Pending...').is_element_displayed():
+            self.errors.append('Pending request is not displayed after request was sent')
+        if self.chat_1.chat_message_input.is_element_displayed():
+            self.errors.append('Input field is displayed despite request has not been accepted yet')
+
+        self.home_2.just_fyi('Accepting of a new contact request')
+        self.home_2.handle_contact_request(self.default_username_1)
+        chat_2 = self.home_2.get_chat_view()
+
+        self.home_2.just_fyi('Verify request acceptor can send messages to request sender after acceptance of cr')
+        message_from_receiver = 'Message from user who has accepted contact request'
+        chat_2.send_message(message_from_receiver)
+        if not self.chat_1.chat_element_by_text(message_from_receiver).is_element_displayed():
+            self.errors.append('Message from accepted user has not been received')
+
+        self.home_1.just_fyi('Verify chat input field has appeared after contact request has been accepted')
+        if not self.chat_1.chat_message_input.is_element_displayed():
+            self.drivers[0].fail('Chat input field has not appeared after contact request has been accepted')
+
+        self.home_1.just_fyi('Verify request sender can send messages to request acceptor after acceptance of cr')
+        self.chat_1.just_fyi("Clicking send button to clear input field from remained request message. Bug #13610")
```

Comment:
Better comment with #TODO : - they are reviewed periodically and easily can be found in code (built-in feature of PyCarm)

Reply:
added #TODO comment